src/scss/components/ec-grid/ec-grid.story.js

```diff
@@ -0,0 +1,175 @@
+
+      <h1 class="grid-story__title">Various column combinations</h1>
+      <div class="ec-grid">
+        <div class="ec-grid__row grid-story__container">
+          <div class="ec-col-2 grid-story__column"><p>.ec-col-2</p></div>
+          <div class="ec-col-4 grid-story__column"><p>.ec-col-4</p></div>
+          <div class="ec-col-6 grid-story__column"><p>.ec-col-6</p></div>
+        </div>
+      </div>
+    `,
+  }))
+  .add('columns', () => ({
+    template: `
+    <div class="grid-story">
+      <h1 class="grid-story__title">12 Columns - 24 pixel gutters</h1>
+      <h2 class="grid-story__title">Grandparent: "ec-grid", Parent: "ec-grid__col" Child: "ec-col-1 to ec-12"</h2>
+      <div class="ec-grid">
+        <div class="ec-grid__col grid-story__container">
+          <div
+            v-for="i in 12"
+            class="ec-col-1 grid-story__column">
+            <p>.ec-col-1 <div>{{i}}</div></p>
+          </div>
+        </div>
+      </div>
+    </div>
+    `,
+  }))
+  .add('nested', () => ({
+    template: `
+    <div class="grid-story">
+      <h1 class="grid-story__title">.ec-grid</h1>
+
+      <div class="ec-grid grid-story__container">
+        <div class="ec-grid__row">
+          <div class="ec-col-6 grid-story__column">
+            <h1 style="font-size: 20px;">.ec-col-6</h1>
+            <div style="border:1px solid black;padding:5px;margin:5px;">
+              <h1 style="font-size: 20px;">.ec-grid</h1>
+
+                <div class="ec-grid">
+                  <div class="ec-grid__row">
+                    <div
+                      v-for="i in 3"
+                      class="ec-col-4 grid-story__column">
+                      <p>.ec-col-4</p>
+                    </div>
+                  </div>
+                </div>
+            </div>
+          </div>
+
+          <div class="ec-col-6 grid-story__column">
+            <h1 style="font-size: 20px;">.ec-col-6</h1>
+            <div style="border:1px solid black;padding:5px;margin:5px;">
+            <h1 style="font-size: 20px;">.ec-grid</h1>
+            <div class="ec-grid">
+              <div class="ec-grid__col">
+                <div
+                  v-for="i in 3"
+                  class="ec-col-4 grid-story__column">
+                  <p>.ec-col-4</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    `,
+  }))
+  .add('responsive', () => ({
+    template: `
+    <div class="grid-story">
+      <h1 class="grid-story__title">Resize to swap between column and row @xs @sm @md @lg @xl breakpoints</h1>
+      <h2 class="grid-story__title">Grandparent: ec-grid</h2>
+      <h2 class="grid-story__title"> Parent: ec-grid__col ec-grid__row-xs ec-grid__col-sm ec-grid__row-md ec-grid__col-lg ec-grid__row-xl</h2>
+
+      <div class="ec-grid grid-story__container">
+        <div class="ec-grid__col ec-grid__row-xs ec-grid__col-sm ec-grid__row-md ec-grid__col-lg ec-grid__row-xl">
```

Comment:
Second opinion wanted: Are we sure we don't want to use `@` for specifying the breakpoint? e.g.
```
<div class="ec-grid__col ec-grid__row@xs ec-grid__col@sm ec-grid__row@md ec-grid__col@lg ec-grid__row@xl">
  <div class="ec-grid__col-6@lg ec-grid__col-9@xs">
```
so it's more visible what's happening.